app/scripts/views/tooltip.js

```diff
@@ -27,6 +27,14 @@ define([
       this.dismissible  = options.dismissible || false;
       this.extraClassNames = options.extraClassNames || '';
 
+      // Event codes for the keyboard keys.
+      this.DOWN_ARROW_KEY_CODE = 40;
```

Comment:
Sounds good to me! I filed https://github.com/mozilla/fxa-content-server/issues/3163 as a follow on.